app/src/main/java/no/usn/mob3000/App.kt

```diff
@@ -604,14 +606,15 @@ fun TopNavbar(
  * @author frigvid
  * @created 2024-10-02
  */
+
```

Comment:
Remember spacing. Do not introduce previously non-existent spacing issues, please.